### PR TITLE
style qualification lists in campus edit form

### LINKED ISSUE #268: Ugly / inconsistent padding and presentation
image (2)

Make it cleaner and nice for the eyes.

### PR DESCRIPTION
**Pull request by boney-bun**
fix #268

### CODE REVIEW

**Comment by boney-bun** [comment]
@pierrealixt let me know if this is good enough:
ford3#279

**Comment by timlinux** [comment]
The number next to the qualification still makes the entry look ugly. Can we have it as a separate column on the right of the text rather and in a pill style?

image

**Comment by Christiaanvdm** [comment]
> The number next to the qualification still makes the entry look ugly. Can we have it as a separate column on the right of the text rather and in a pill style?

I would then also recommend adding a header "Course" and "SAQA Website"

**Comment by boney-bun** [comment]
@timlinux i'm having trouble to align the saqa list if it's on the right since the length of qualifications are varied. so i put them in the left instead.
would this be ok?
ford3#279_rev

@Christiaanvdm i'm not sure about *adding a header*. 
if this means to add more explanations to the qualifications, i don't think we will have enough space.

File: django_project/ford3/static/css/campus_form.css [hunk 1]
```diff
@@ -92,3 +94,22 @@ button.edu-button-blue {
     position: relative;
     top: -0.25rem;
 }
+
+h5.text-center.edu-underline {
+    padding-top: 0.5rem;
+    padding-bottom: 0.5rem;
+}
+
+.spacer {
```
Comment (pierrealixt) [inline django_project/ford3/static/css/campus_form.css:103]: let's use Bootstrap spacing class methods: https://getbootstrap.com/docs/4.0/utilities/spacing/

File: django_project/ford3/static/css/campus_form.css [hunk 2]
```diff
@@ -92,3 +94,22 @@ button.edu-button-blue {
...
+    margin-top: 5px;
+    margin-bottom: 5px;
+}
+
+.nounderline {
+    text-decoration: none !important;
+}
+
+.nomlr {
```
Comment (pierrealixt) [inline django_project/ford3/static/css/campus_form.css:112]: same here https://getbootstrap.com/docs/4.0/utilities/spacing/

**Review by pierrealixt — COMMENTED** [review]
It's better but it still doesn't look that gorgeous.
I quickly ask Carina for a review.

**Comment by pierrealixt** [comment]
> i'm having trouble to align the saqa list if it's on the right since the length of qualifications are varied

You should use Boostrap class methods. You could use something like this:
```
<div class='container'>
  <div class='row'>
    <div class='col-md-10'>
      Bachelor of Arts
    </div
   <div class='col-md-2'>
     <a href=''>40505</a>
   </div>
  </div>
</div>
```

**Comment by pierrealixt** [comment]
I'll take care of this in #325

**Pull request closed without merging**